### DISCUSSION TITLE
manually change /web/view/views.py to set vehicle[1]

### DISCUSSION THREAD

**Discussion by frozengpu** [flobz/psa_car_controller#230] [Q&A]
Hi all, I have two vehicles in my psa account and want the dashboard to show only vehicle_list[1] data. 
How can I change the used vehicle in psa_car_controller. I found in the source code the /web/view/views.py where I can change 
`car = CONFIG.myp.vehicles_list[0]  # todo handle multiple car`, 
but I don't know where to find the correct spot in my home assitant installation. 

Do you have any hints?
Thanks!
BR, Tim

**Comment by flobz** [comment]
Well, as you can see the features isn't developed. 
Solutions are  : 
- Edit the line you mention 
- Use an environment variable to set witch car to use. 
- Or handle multiple car in the web ui.

**Reply by frozengpu** [reply]
Thanks for the options. I am fairly new to all this and have some entry points now. I think I'll try the path of editing the py file first.
